.github/workflows/rust.yml

```diff
@@ -3,28 +3,58 @@ name: Rust
 on: [push]
 
 jobs:
-  build:
+
+  codestyle:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Rust
+        uses: hecrj/setup-rust-action@v1
+        with:
+          components: rustfmt
+          rust-version: nightly
+      - uses: actions/checkout@v2
+      - run: cargo fmt -- --check --config-path <(echo 'license_template_path = "HEADER"')
+
+  lint:
```

Comment:
Is clippy linting a separate step for [this](https://github.com/andygrove/sqlparser-rs/blob/d32df527e68dd76d857f47ea051a3ec22138469b/.travis.yml#L51) reason?

Reply:
Actually only incidentally: it's primarily so the jobs can be run in parallel. I can add a comment to that effect if that's in fact still an issue. (It's also just me copying what other Rustaceans seem to be doing. 🙂)

Reply:
Again, as a prerequisite for the `test` step, I don't think it's making things faster.

Sorry about making so many comments; I'm fine with narrowing the scope of this PR to just the code coverage, if you prefer!

Reply:
That's fine with me. I'll remove the job dependencies from the `test` job.

Reply:
I removed it as a prereq to `test`.